examples/opensora_hpcai/scripts/inference.py

```diff
@@ -33,10 +32,16 @@
 logger = logging.getLogger(__name__)
 
 
-def init_env(mode, device_target, enable_dvm=False):
+def init_env(mode, device_target, enable_dvm=False, debug: bool = False):
+    if debug and mode == ms.GRAPH_MODE:  # force PyNative mode when debugging
+        logger.warning("Debug mode is on, switching execution mode to PyNative.")
+        mode = ms.PYNATIVE_MODE
+
     ms.set_context(
         mode=mode,
         device_target=device_target,
+        gpu_config={"matmul_allow_tf32": True},
```

Comment:
is it necessary?

Reply:
Not strictly necessary, just to align with [OpenSora repo](https://github.com/hpcaitech/Open-Sora/blob/c6cc021d612456455addc1b3164c1d43eeb33b97/scripts/inference-long.py#L149). Even without this setting results are visually good. Should I remove it or add a CLI argument for this?

Reply:
better to remove for keeping stable